crates/re_types_builder/src/codegen/rust/api.rs

```diff
@@ -355,6 +355,47 @@ fn quote_struct(
 
     let quoted_builder = quote_builder_from_obj(obj);
 
+    let quoted_heap_size_bytes = if obj
+        .fields
+        .iter()
+        .any(|field| field.has_attr(crate::ATTR_RUST_SERDE_TYPE))
+    {
+        // TODO(cmc): serde types are a temporary hack that's not worth worrying about.
+        quote!()
+    } else {
+        let heap_size_bytes_impl = if is_tuple_struct_from_obj(obj) {
+            quote!(self.0.heap_size_bytes())
+        } else {
+            let quoted_heap_size_bytes = obj.fields.iter().map(|obj_field| {
+                let field_name = format_ident!("{}", obj_field.name);
```

Comment:
the later incarnation did `crate::to_pascal_case(`. This probably should as well

Reply:
The other does that because it's enumerating enum variants! This one is generating struct field names, so not pascal case.